drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/DTCellValueWidgetFactory.java

```diff
@@ -317,27 +253,55 @@ public Widget getWidget(ActionInsertFactCol52 column,
         //Check if the column has a "Value List" or an enumeration. Value List takes precedence
         if (columnUtilities.hasValueList(column)) {
             final String[] valueList = columnUtilities.getValueList(column);
-            return makeListBox(DropDownData.create(valueList),
-                               column,
-                               value);
+            return makeListBoxForColumn(DropDownData.create(valueList),
+                                        null,
+                                        column,
+                                        value,
+                                        false);
         } else if (oracle.hasEnums(column.getFactType(),
                                    column.getFactField())) {
-            final LimitedEntryDropDownManager.Context context = new LimitedEntryDropDownManager.Context(column);
-            final Map<String, String> currentValueMap = dropDownManager.getCurrentValueMap(context);
-            final DropDownData dd = oracle.getEnums(column.getFactType(),
-                                                    column.getFactField(),
-                                                    currentValueMap);
-            if (dd == null) {
-                return makeListBox(DropDownData.create(new String[0]),
-                                   column,
-                                   value);
-            }
-            return makeListBox(dd,
-                               column,
-                               value);
+            return makeHasEnumsListBox(null,
```

Comment:
Yes I am sure. If you have a look at `LimitedEntryDropDownManager` class, the `pattern` is used just in case when column is `ConditionCol52` or `ActionSetFieldCol52` and this is exactly when I call `Context` constructor with non null pattern. However, I will add some NPE preventing into `LimitedEntryDropDownManager`. Thanks.